pkg/cluster/gitops.go

```diff
@@ -15,29 +15,16 @@ func gitOpsEntry() *ConfigManagerEntry {
 		},
 		Processors: []ParsedProcessor{processGitOps},
 		Defaulters: []Defaulter{
-			func(c *Config) error {
-				if c.GitOpsConfig != nil {
-					c.GitOpsConfig.SetDefaults()
-				}
-				return nil
-			},
+			setGitOpsDefaults,
 			SetDefaultFluxGitHubConfigPath,
+			setFluxDefaults,
+			SetDefaultFluxConfigPath,
 		},
 		Validations: []Validation{
-			func(c *Config) error {
-				if c.GitOpsConfig != nil {
-					return c.GitOpsConfig.Validate()
-				}
-				return nil
-			},
-			func(c *Config) error {
-				if c.GitOpsConfig != nil {
-					if err := validateSameNamespace(c, c.GitOpsConfig); err != nil {
-						return err
-					}
-				}
-				return nil
-			},
+			validateGitOps,
+			validateGitOpsNamespace,
+			validateFlux,
+			validateFluxNamespace,
```

Comment:
Same here, these are already added by the flux entry

Reply:
https://github.com/aws/eks-anywhere/pull/1827/commits/e0c1cea03b3576d9edf4a990155eec4410d6222d